hydra-filters/src/test/java/com/addthis/hydra/data/filter/bundle/TestBundleFilterTimeRange.java

```diff
@@ -13,121 +13,107 @@
-        assertEquals(input, time.substring(2));
+    public void testBeforeNoTimezone() throws IOException {
+        BundleFilterTimeRange filter1 = Configs.decodeObject(BundleFilterTimeRange.class, BEFORE_160923);
+        BundleFilterTimeRange filter2 = Configs.decodeObject(BundleFilterTimeRange.class, BEFORE_150925);
+        assertTrue("Expected true since filtered out later than 160923", filter1.filter(bundle));
+        assertFalse("Expected false since filtered out later than 150925", filter2.filter(bundle));
     }
 
     @Test
-    public void testBundleFilterTimeRangeEST() {
-        DateTimeZone tz = DateTimeZone.forID("America/New_York");
-        DateTime abDateTime = utc.toDateTime(tz);
-        String input = abDateTime.toString("yyMMddHHmm");
-
-        BundleFilterTimeRange bftr = new BundleFilterTimeRange(timeIn, "121201", "130101", true, "yyMMddHHmm", "America/New_York");
-
-        DateTimeFormatter format = bftr.getTimeZoneFormat();
-
-        String time = format.parseDateTime(input).getYear() + "" +
-                df.format(format.parseDateTime(input).getMonthOfYear()) + "" +
-                df.format(format.parseDateTime(input).getDayOfMonth()) + "" +
-                df.format(format.parseDateTime(input).getHourOfDay()) + "" +
-                df.format(format.parseDateTime(input).getMinuteOfHour());
-        assertEquals(input, time.substring(2));
+    public void testAfterNoTimezone() throws IOException {
+        BundleFilterTimeRange filter1 = Configs.decodeObject(BundleFilterTimeRange.class, AFTER_160921);
+        BundleFilterTimeRange filter2 = Configs.decodeObject(BundleFilterTimeRange.class, AFTER_170921);
+        assertTrue("Expected true since filtered out earlier than 160921", filter1.filter(bundle));
+        assertFalse("Expected false since filtered out earlier than 170921", filter2.filter(bundle));
     }
 
     @Test
-    public void testBundleFilterTimeRangePST() {
-        DateTimeZone tz = DateTimeZone.forID("America/Los_Angeles");
-        DateTime abDateTime = utc.toDateTime(tz);
-        String input = abDateTime.toString("yyMMddHHmm");
-
-        BundleFilterTimeRange bftr = new BundleFilterTimeRange(timeIn, "121201", "130101", true, "yyMMddHHmm", "America/Los_Angeles");
-
-        DateTimeFormatter format = bftr.getTimeZoneFormat();
-
-        String time = format.parseDateTime(input).getYear() + "" +
-                df.format(format.parseDateTime(input).getMonthOfYear()) + "" +
-                df.format(format.parseDateTime(input).getDayOfMonth()) + "" +
-                df.format(format.parseDateTime(input).getHourOfDay()) + "" +
-                df.format(format.parseDateTime(input).getMinuteOfHour());
-        assertEquals(input, time.substring(2));
+    public void testMinusBefore() throws IOException {
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, "time:TIME, before:-20160923, timeFormat:YYYYMMdd");
```

Comment:
`before:-20160923, timeFormat:YYYYMMdd` is misleading as the reader may think the value for `before` should match `timeFormat`, while in reality a value starting with a `-` indicates number of days and `timeFormat` is not used to parse it. Something like `before:-10000` would be better.